WordPress/Classes/ViewRelated/Views/List/ListTableViewCell.swift

```diff
@@ -14,6 +14,9 @@ class ListTableViewCell: UITableViewCell, NibReusable {
 
     // MARK: Properties
 
+    @objc static let reuseIdentifier = defaultReuseID
```

Comment:
Same nitpick/request as above - can a comment be added here indicating `reuseIdentifier` can be removed when the VCs are converted to Swift? It'd be nice to _eventually_ use `defaultReuseID` directly.

Reply:
Yup! Agreed. 😄  Will add docs here too.